packages/framework-info/src/context.js

```diff
@@ -1,30 +1,41 @@
 import { cwd, version as nodejsVersion } from 'process'
 
-import isPlainObj from 'is-plain-obj'
 import { locatePath } from 'locate-path'
 import { readPackageUp } from 'read-pkg-up'
 
-export const getPackageJson = async (projectDir) => {
+/*
```

Comment:
can we migrate it to typescript?

Reply:
No, the files in `src` are directly used in node, see https://github.com/netlify/build/blob/main/packages/framework-info/package.json#L8

So we need to add another build step.

Reply:
I have a branch where I have most of it setup, what is missing is ava -> vitest migration as ava cannot handle ts/esm